main.tex

```diff
@@ -121,8 +121,7 @@
 \pagenumbering{roman}
 
 \begin{abstract}
-    \noindent
-    \lipsum[3]
+Co-operative survival games refer to a subset of political choice games wherein `players' must work together to overcome disaster, else suffer the consequences through both personal and communal damage. Furthermore, without the ability to monitor a system and in the absence of a centralised authority, it becomes impossible to form enduring self-governing institutions, resulting in the proposed self-organising mechanisms for `solving' such games potentially functioning in harmful ways. This project utilises a self-organising, multi-agent system to simulate an iterated collective action, co-operative survival game comprising a resource management problem, the `solution' to which involves a system of localised governance. Through a series of experiments where the behaviour of the players in the system is increasingly randomised and the presence of self-organising mechanisms is toggled, we investigate the causes of instability. Ultimately, we conclude that, through the integration of governance using treaties, it is possible for such a system to effectively self-organise to minimise the total deaths of the players involved.
```

Comment:
```suggestion
Co-operative survival games refer to a subset of political choice games wherein `players' must work together to overcome disaster, or else suffer the consequences through both personal and communal damage. Furthermore, without the ability to monitor a system and in the absence of a centralised authority, it becomes impossible to form enduring self-governing institutions, resulting in the proposed self-organising mechanisms for `solving' such games potentially functioning in harmful ways. This project utilises a self-organising multi-agent system to simulate an iterated collective action, co-operative survival game comprising a resource management problem, the `solution' to which involves a system of localised governance. Through a series of experiments where the behaviour of the players in the system is increasingly randomised and the presence of self-organising mechanisms is toggled, we investigate the causes of instability. Ultimately we conclude that through the integration of governance using treaties, it is possible for such a system to effectively self-organise to minimise the total deaths of the players involved.
```

Reply:
using a comma before "disaster, else" negates the need for an "or"

also drop-in-clause would be needed if we're keeping "using treaties, it is" or else we remove both

Reply:
Overall definitely agree with your second change though, made a slight change :)